### ISSUE TITLE
Separate nearby points on map surveys

### ISSUE THREAD

**Issue by jlegrone** [Bernie-2016/ground-control#427]
Right now multiple events show up right on top of one another, and are hard or impossible to click. We could implement a solution from https://developers.google.com/maps/articles/toomanymarkers

**Comment by nkammah** [comment]
Code [here](https://github.com/Bernie-2016/ground-control/blob/master/src/frontend/components/survey-renderers/PhonebankRSVPSurvey.js)

**Comment by saikat** [comment]
@nkammah yes, that is correct (sorry wasn't sure if you were asking or pointing it out, but either way you are right!).
